internal/bootstrap/bootstrap_tpl.go

```diff
@@ -112,10 +112,21 @@ type BootstrapTplArgs struct {
 	// the envoy_prometheus_bind_addr listener.
 	PrometheusScrapePath string
 
-	PrometheusCAFile   string
-	PrometheusCAPath   string
+	// PrometheusCAFile is the path to a CA file for Envoy to use when serving TLS on the Prometheius metrics
+	// endpoint. Only applicable when envoy_prometheus_bind_addr is set in the proxy config.
```

Comment:
This file is generated by `make copy-bootstrap-config` so these comments will be lost when re-generating the file. Basically, this is code copied over from the Consul repo.

I like these comments, so we could add them to the [corresponding file](https://github.com/hashicorp/consul/blob/5e156772f6a7fba5324eb6804ae4e93c091229a6/command/connect/envoy/bootstrap_tpl.go#L114-L117) in consul?

Reply:
I can do that!

Reply:
https://github.com/hashicorp/consul/pull/15064